react/UnitPrice.tsx

```diff
@@ -12,10 +12,22 @@ const CSS_HANDLES = [
   'unitPriceContainer',
   'unitPricePerUnitCurrency',
   'unitPriceMeasurementUnit',
+  'unitListPrice',
 ] as const
 
-const UnitPrice: StorefrontFunctionComponent<TextAlignProp> = ({
+interface UnitPriceProps extends TextAlignProp {
+  unitPriceDisplay: UnitPriceDisplayType
+  displayUnitListPrice: DisplayUnitListPriceType
+}
+
+type UnitPriceDisplayType = 'always' | 'default'
+
+type DisplayUnitListPriceType = 'always' | 'never'
```

Comment:
I think `always` here is a bit misleading. You won't always show the listPrice, only when they prices are different. Maybe a better name for this is `default` like you did above.
```suggestion
type DisplayUnitListPriceType = 'default' | 'never'
```